static/src/javascripts/lib/load-script.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+const loadScript = (src: string, props: Object): Promise<any> => {
+    if (typeof src !== 'string') {
```

Comment:
Maybe we should a comment here, remembering us of the fact, that we could remove this check after the ES6 conversion, since flow doesn't allow an empty `src`